cucumber/kubernetes/features/pki.feature

```diff
@@ -1,8 +1,16 @@
 Feature: Certificates issued by the login method.
 
-  Scenario: The subject name is the dot-separated relative Conjur host ID.
```

Comment:
Does this mean our old behavior is changing?

Reply:
yes but the change is affected also in the `/authenticate` request that follows this response. 

The client sends a CSR in the `/inject_client_cert` request. This CSR is signed and is sent back to the client. After that the certificate is sent again to the server in the `/authenticate` request and is validated.

We changed the CSR that is returned at the end of the first request, _and_ the way it is treated in the second request so it is seamless to the client - old or new.